lib/experimental/webfonts.php

```diff
@@ -227,3 +313,61 @@ function( $mime_types ) {
 		return $mime_types;
 	}
 );
+
+/*
+ * Deprecated functions provided here to give extenders time to change
+ * their plugins/themes before this API is introduced into Core.
+ *
+ * BACKPORT NOTE: Do not backport these deprecated functions to Core.
+ */
+
+if ( ! function_exists( 'wp_enqueue_webfonts' ) ) {
+	/**
+	 * Enqueues a collection of font families.
+	 *
+	 * Example of how to enqueue Source Serif Pro and Roboto font families, both registered beforehand.
+	 *
+	 * <code>
+	 * wp_enqueue_webfonts(
+	 *  'Roboto',
+	 *  'Sans Serif Pro'
+	 * );
+	 * </code>
+	 *
+	 * Font families should be enqueued from the `init` hook or later.
+	 *
+	 * BACKPORT NOTE: Do not backport this function.
+	 *
+	 * @since 6.0.0
+	 * @deprecated 6.1.0 Use wp_enqueue_webfont().
+	 *
+	 * @param string[] $font_families Font family handles (array of strings).
+	 */
+	function wp_enqueue_webfonts( array $font_families ) {
+		_deprecated_function( __FUNCTION__, '6.1.0', 'wp_enqueue_webfont()' );
```

Comment:
```suggestion
		_deprecated_function( __FUNCTION__, 'X.X.X', 'wp_enqueue_webfont()' );
```

Reply:
I know this is nit-picking, but just want to make sure when this gets merged we have the correct version numbers. 👍